cmd/autoowners/main.go

```diff
@@ -133,40 +136,79 @@ func (orgRepo *orgRepo) getOwnersHTTP() (err error) {
 		data, err := gc.GetFile(orgRepo.Organization, orgRepo.Repository, filename, "HEAD")
 		if err != nil {
 			if _, nf := err.(*github.FileNotFound); nf {
+				logrus.WithField("orgRepo", fmt.Sprintf("%s/%s", orgRepo.Organization, orgRepo.Repository)).
+					WithField("filename", filename).
+					Debug("Not found file in the upstream repo")
+				orgRepo.OwnersFileExists = false
 				continue
 			} else {
 				return err
 			}
 		}
 
-		var target interface{}
+		orgRepo.OwnersFileExists = true
 		switch filename {
 		case "OWNERS":
-			target = &orgRepo.Owners
+			var target interface{}
```

Comment:
Feels like we should be able to use an upstream library to do the loading?

Reply:
Tried ... but did not go through. :(